src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -2309,4 +2374,74 @@ private static int log2(int value)
         }
         #endregion
     }
+
+    /// <summary>
+    /// A class that generates an IEqualityComparer for this SortedSet. Requires that the definition of
+    /// equality defined by the IComparer for this SortedSet be consistent with the default IEqualityComparer
+    /// for the type T. If not, such an IEqualityComparer should be provided through the constructor.
+    /// </summary>    
+    internal class SortedSetEqualityComparer<T> : IEqualityComparer<SortedSet<T>> 
+    {
+        private IComparer<T> comparer;
+        private IEqualityComparer<T> e_comparer;
```

Comment:
Nit: These fields can be `readonly` and should match the naming guidelines, e.g. `_comparer` and `_memberEqualityComparer`.